public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -824,6 +824,14 @@ export class LokiDatasource
   getQueryHints(query: LokiQuery, result: DataFrame[]): QueryHint[] {
     return getQueryHints(query.expr, result);
   }
+
+  getDefaultQuery(app: CoreApp): Partial<LokiQuery> {
```

Comment:
Could we do `LokiQuery` instead of `Partial<LokiQuery>` and add default values for all required fields which is I think only `expr`. And also create a defaultQuery for app that is not `UnifiedAlerting`?

Reply:
Yep – I think that's sensible 👍  Will apply full defaults for both Loki and Prometheus for consistency.